setup.py

```diff
@@ -14,7 +14,7 @@
         'flake8',
     ],
     install_requires=[
-        'requests',
```

Comment:
Since we're doing a library here, can we use >= or <= instead of hard pinning (it's a pain if you have two libs hard pinning dependencies). What's holding us on this particular requests version actually ?

(Typo in commit message as well: Ping -> Pin)

Reply:
requests updated one of its internal lib which is not py3.5-supported: https://github.com/Picterra/picterra-python/runs/7392958658?check_suite_focus=true

commit message typo fixed

Reply:
Ah right. I feel we could then require python 3.7 (Released in 2018) as it seems like the minimal version everyone is moving towards.

But maybe let's do another MR for that as it requires changing the python on the CI as well.

Reply:
yep we can open an issue for that

Reply:
Anything left to do here?

Reply:
I've put the "<=" and opened an issue to just require py 3.7

Reply:
#72